src/main/java/de/dennisguse/opentracks/content/data/TrackPointsColumns.java

```diff
@@ -31,6 +31,7 @@
     String TABLE_NAME = "trackpoints";
     Uri CONTENT_URI_BY_ID = Uri.parse(ContentProviderUtils.CONTENT_BASE_URI + "/" + TABLE_NAME);
     Uri CONTENT_URI_BY_TRACKID = Uri.parse(ContentProviderUtils.CONTENT_BASE_URI + "/" + TABLE_NAME + "/trackid");
+    Uri CONTENT_URI_SENSOR_STATS = Uri.parse(ContentProviderUtils.CONTENT_BASE_URI + "/" + TABLE_NAME + "/sensorstats");
```

Comment:
I would consider this computation rather part of the track than of the trackpoints.
Technically, you are completely right as the data is derived from the trackpoints.
However, in the track-table, we have all the other statistics - only difference is that those are precomputed.